### ISSUE TITLE
[Bug]: Invalid response: 423 Locked

### ISSUE THREAD

**Issue by TheIcyStar** [remotely-save/remotely-save#792]
### What happened?

I began getting some sync problems with this error message: "Invalid Response: 423 Locked". It keeps failing on the same folder path, but not on a specific file.
![image](https://github.com/user-attachments/assets/228373e4-e2aa-4440-9ff1-c310fd433512)

Only my windows desktop is having this sync issue, all of my other devices are able to sync just fine. I did move some folders with files into other folders recently, but only my windows desktop is stuck in the past now. I have also used [PowerToys File Locksmith](https://learn.microsoft.com/en-us/windows/powertoys/file-locksmith) to check if there is some process locking any folders, and there appears to be no processes locking anything.

The console has this error logged:
```
AggregateError: 
    Error: (folder path omitted): Invalid response: 423 Locked
        at eval (plugin:remotely-save:229:46276)
    at eval (plugin:remotely-save:229:46436)
    at Generator.next (<anonymous>)
    at s (plugin:remotely-save:76:371388)
eval @ plugin:remotely-save:236

```

Any help will be greatly appreciated!

### What OS are you using?

Windows

### What remote cloud services are you using? (Please choose the specified one if it's in the list)

webdav (NextCloud)

### Version of the plugin

0.5.18

### Version of Obsidian

1.6.7

### Using password or not

- [ ] Yes.

### Ensure no sensitive information

- [X] I ensure that no sensitive information is submitted in the issue.

**Comment by fyears** [comment]
please export you sync plan to check if anything is abnormal.

alternatively, you can create a test vault and create some non-empty test files to see if the error can be reproduced

**Comment by TheIcyStar** [comment]
I wasn't too sure what to look for in the sync plan, as it looked similar enough when comparing what I had on the broken vs working machines...

However **I figured out how to fix the issue**:
1. In Remotely Save's settings, I set the sync direction to _Incremental Pull And Delete_
2. Did a manual sync
3. Deleted the old duplicate folders (as I now had both the old, pre-moved folders and the new moved folders)
4. Set the sync direction back to normal (bidirectional)
5. (Optional) verified I'm not missing anything by setting "Abort sync if modification above percentage" to something super low, and doing a manual sync with that

My guess is that some state was messed up after moving some folders into other folders, and nextcloud was getting angry for obsidian requesting something that wasn't there? Anyways, looks like it's all good now. Thanks for the suggestions and for your work on this plugin!
